src/Components/Aspire.Azure.Security.KeyVault/Aspire.Azure.Security.KeyVault.csproj

```diff
@@ -6,7 +6,7 @@
     <PackageTags>$(ComponentAzurePackageTags) keyvault secrets</PackageTags>
     <Description>A client for Azure Key Vault that integrates with Aspire, including health checks, logging and telemetry.</Description>
     <PackageIconFullPath>$(SharedDir)AzureKeyVault_256x.png</PackageIconFullPath>
-    <NoWarn>$(NoWarn);SYSLIB1100;SYSLIB1101</NoWarn>
+    <NoWarn>$(NoWarn);SYSLIB1100;SYSLIB1101;IDE0200</NoWarn>
```

Comment:
We shouldn't suppress this globally, but just in the specific places that need to suppress it.